pallene/translator.lua

```diff
@@ -61,6 +61,18 @@ function Translator:erase_region(start_index, stop_index)
     self.last_index = stop_index + 1
 end
 
+-- This is a workaround to handle math.log built-in optimizations.
+-- Because Pallene currently cannot handle optional parameters,
+-- we've decided to introduce math.ln as a replacement for single param log(x).
+-- But for --emit-lua, we must do something to make the code work in pure Lua.
+-- For now, the easiest thing to do is inject math.ln = math.log at the top.
+-- A smarter routine would replace math.ln with math.log.
```

Comment:
I love what this comment is saying, but I would expect that the translation.lua might not be the first place someone would look for it. What do you think about moving it to the  coder.lua or builtins.lua?